docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  autorest-gen: 
+    image: autorest-gen
+    # build: .
+    environment: 
+      ENV_YML_FILE_URL: # YML PATH
+      ENV_OUTPUT_PATH: "/app/output"
+      ENV_NAMESPACE: "Agoda.BookingApi.GenClient"
```

Comment:
is this a sample/template file?

Reply:
Yes. Will update and parametrize it from teamcity in the next PR